proposals/lotus-light.md

```diff
@@ -0,0 +1,87 @@
+# Lotus Light Mode
+
+Authors: @magik6k
+
+Initial PR: TBD <!-- Reference the PR first proposing this document. Oooh, self-reference! -->
+
+## Purpose &amp; impact
+#### Background &amp; intent
+
+Currently, running lotus nodes requires dedicating lots of resources. Users should
+be able to talk to the blockchain without validating it fully
+
+#### Assumptions &amp; hypotheses
+* Filecoin chain can be followed just by validating blockheaders with bits of previous state safely
+* Bitswap is fast enough to support this
+
+#### User workflow example
+Instead of syncing a full node from a snapshot (big, ~90G), the client would
+start a light node, which would trust state in blockheaders, and follow the chain
+just by running basic validation on blockheaders (checking miner power/signatures
+using older state we already trust)
+
+#### Impact
+Lotus would be much easier to run for non-miner users
+
+#### Leverage
+_How much would nailing this project improve our knowledge and ability to execute future projects?_
```

Comment:
```suggestion
The current difficulty of running a Lotus node is a frequent painpoint for developers and users. The typical solution is to abstract Lotus away behind some service, but we stand to gain a lot from also lowering the barrier to entry to the Filecoin network itself. This would reduce the space reqs needed to be connected to the network (no more running out of disk space every few weeks), the time to get started (no more gigabytes of data to download), and the computing power needed (no more hearing your laptop make angry sounds when it runs Lotus).
```